x-pack/plugins/observability/public/pages/slo_edit/components/common/data_preview_chart.tsx

```diff
@@ -7,7 +7,8 @@
 
 import {
```

Comment:
I've edited this file for my own personal proof of concept work. 

The final PR will utilize Kevin Delemme's new event bar chart.